### ISSUE TITLE
Date validation

### ISSUE THREAD

**Issue by inacior** [carloshpds/worktime#11]
## Problem
When we passes a incorrect date on `-d` arg, the project does not validate.

## Example
`my-worktime check -u $USER& -p $PASS$ -c $EMPLOYEE% -j 08:48 -d 123456`
Error:
```
⠙ Buscando dados no Ahgora
REQUEST ERROR: empty_month
✖ Não há nenhuma batida para esta data ainda.
```

or
`my-worktime check -u $USER& -p $PASS$ -c $EMPLOYEE% -j 08:48 -d 28-09-2020`
Error:
```
Deprecation warning: value provided is not in a recognized RFC2822 or ISO format. moment construction falls back to js Date(), which is not reliable across all browsers and versions. Non RFC2822/ISO date formats are discouraged and will be removed in an upcoming major release. Please refer to http://momentjs.com/guides/#/warnings/js-date/ for more info.
Arguments:
[0] _isAMomentObject: true, _isUTC: false, _useUTC: false, _l: undefined, _i: 28-09-2020, _f: undefined, _strict: undefined, _locale: [object Object]
Error
    at Function.createFromInputFallback (/Users/rinacio/.nvm/versions/node/v8.13.0/lib/node_modules/my-worktime/node_modules/moment/moment.js:319:25)
    at configFromString (/Users/rinacio/.nvm/versions/node/v8.13.0/lib/node_modules/my-worktime/node_modules/moment/moment.js:2536:19)
    at configFromInput (/Users/rinacio/.nvm/versions/node/v8.13.0/lib/node_modules/my-worktime/node_modules/moment/moment.js:2978:13)
    at prepareConfig (/Users/rinacio/.nvm/versions/node/v8.13.0/lib/node_modules/my-worktime/node_modules/moment/moment.js:2961:13)
    at createFromConfig (/Users/rinacio/.nvm/versions/node/v8.13.0/lib/node_modules/my-worktime/node_modules/moment/moment.js:2928:44)
    at createLocalOrUTC (/Users/rinacio/.nvm/versions/node/v8.13.0/lib/node_modules/my-worktime/node_modules/moment/moment.js:3022:16)
    at createLocal (/Users/rinacio/.nvm/versions/node/v8.13.0/lib/node_modules/my-worktime/node_modules/moment/moment.js:3026:16)
    at hooks (/Users/rinacio/.nvm/versions/node/v8.13.0/lib/node_modules/my-worktime/node_modules/moment/moment.js:16:29)
    at CheckCommand.<anonymous> (/Users/rinacio/.nvm/versions/node/v8.13.0/lib/node_modules/my-worktime/lib/commands/check.js:35:49)
    at Generator.next (<anonymous>)
⠇ Buscando dados no Ahgora
REQUEST ERROR: empty_month
✖ Não há nenhuma batida para esta data ainda.
```

**Comment by carloshpds** [comment]
@renaninacio Thanks for your report! I'm gonna create a date validator to this field, but if you want to use the `date` param, you have to pass using the pattern `YYYY-MM-DD` by now

**Comment by carloshpds** [comment]
@renaninacio Date validation is available in `1.1.0`
